### PR TITLE
feat: Add explanations when to use odrl:target

### PR DESCRIPTION
**Pull request by sebbader-sap**
* adjust schemas
* adjust readmes for the catalog and negotiation sections

Solves #197

### CODE REVIEW

File: catalog/catalog.protocol.md [hunk 1]
```diff
@@ -119,9 +123,9 @@ A [Dataset](../model/terminology.md#dataset) is a [DCAT Dataset](https://www.w3.
...
-target attributes. The target of an [Offer](../model/terminology.md#offer) is the associated [Dataset](../model/terminology.md#dataset).
+explicit `target` attributes. The `target` of an [Offer](../model/terminology.md#offer) is the associated [Dataset](../model/terminology.md#dataset). This is in line with the semantics of `hasPolicy` as defined in the [ODRL Information Model](https://www.w3.org/TR/odrl-model/#policy-has), explaining that the subject (here the Dataset) is automatically the `target` of each Rule. To prevent conflicts, the `target` attribute must not be set explicitely, neither in the Offer itself or in its Rules. 
 
-> Note: As `odrl:hasPolicy rdfs:domain odrl:Asset`, each [Dataset](../model/terminology.md#dataset) is also an `odrl:Asset` from an ODRL perspective.
+> Note: As ODRL defines that `odrl:hasPolicy rdfs:domain odrl:Asset`, each [Dataset](../model/terminology.md#dataset) is also an `odrl:Asset` from an ODRL perspective.
```
Comment (ssteinbuss) [inline catalog/catalog.protocol.md:128]: Consider removing this comment

Comment (matgnt) [reply catalog/catalog.protocol.md:128]: Agree. It is a strong statement and would need a reference.

**Review by matgnt — APPROVED** [review]
With the small change mentioned, I'm approving the rest.

**Review by jimmarino — APPROVED** [review]

File: catalog/catalog.protocol.md (hunk 1 above)
Comment (matgnt) [reply catalog/catalog.protocol.md:128]: will be done in a separate PR

**Pull request merged**